internal/pkg/pkg_test.go

```diff
@@ -129,18 +129,18 @@ kind: Custom
 spec:
   image: nginx:1.2.3`,
 				`
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
 kind: Kptfile
 metadata:
   name: mysql
-setterDefinitions:
-  replicas:
-    description: "replica setter"
-    type: integer
-setterValues:
-  replicas: 5`,
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/apply-setters:unstable
+      configMap:
+        replicas: 5
+`,
 				`
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1alpha2
 kind: Pipeline
```

Comment:
we don't have a `Pipeline` kind anymore, maybe file an issue for clean up here?

Reply:
sorry merged this before seeing this comment.  We should do a cleanup overall.